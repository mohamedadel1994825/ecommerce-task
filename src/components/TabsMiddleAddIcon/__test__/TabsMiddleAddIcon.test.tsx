import { cleanup, render } from "@testing-library/react-native";
import React from "react";
import { TabsMiddleAddIcon } from "../index";

describe("TabsMiddleAddIcon", () => {
  afterEach(cleanup);

  it("should match snapshot", () => {
    const { toJSON } = render(<TabsMiddleAddIcon />);
    expect(toJSON()).toMatchSnapshot();
  });
});
