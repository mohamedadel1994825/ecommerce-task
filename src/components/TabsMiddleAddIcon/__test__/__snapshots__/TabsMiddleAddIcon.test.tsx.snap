// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabsMiddleAddIcon should match snapshot 1`] = `
<View
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    {
      "alignItems": "center",
      "alignSelf": "center",
      "backgroundColor": "#0d0061",
      "borderRadius": 100,
      "bottom": 20,
      "height": 90,
      "opacity": 1,
      "position": "absolute",
      "width": 90,
      "zIndex": 100,
    }
  }
>
  <Text
    allowFontScaling={false}
    selectable={false}
    style={
      [
        {
          "color": "#FFFFFF",
          "fontSize": 24,
        },
        {
          "marginTop": 18,
        },
        {
          "fontFamily": "Material Design Icons",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        {},
      ]
    }
  >
    Û∞êï
  </Text>
  <Text
    style={
      {
        "color": "#FFFFFF",
        "fontSize": 12,
        "marginTop": 10,
      }
    }
  >
    navigation:deals:headerTitle
  </Text>
</View>
`;
